riak_repl/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG - Riak_Repl
 
+## 1.0.1
+
+ * [FIX] add a condition to check that `rt_sink_connected_to` is a dict before collecting `sink_stats`
```

Comment:
```suggestion
 * [FIX] Add a condition to check that `rt_sink_connected_to` is a dict before collecting `sink_stats`
```
A small nit to update capitalization

Reply:
If I cap line 5 I'll be compelled to fix the other lines so they are congruent... thoughts?

Reply:
Up to you, either way is fine